pom.xml

```diff
@@ -115,24 +117,16 @@
         <groupId>org.eluder.coveralls</groupId>
         <artifactId>coveralls-maven-plugin</artifactId>
         <version>4.3.0</version>
+        <configuration>
+          <sourceDirectories>
+            <sourceDirectory>${basedir}/src/main/java</sourceDirectory>
+          </sourceDirectories>
+        </configuration>
       </plugin>
       <plugin>
         <groupId>org.jacoco</groupId>
         <artifactId>jacoco-maven-plugin</artifactId>
-        <version>0.8.1</version>
-        <executions>
-          <execution>
-            <id>prepare-agent</id>
-            <goals>
-              <goal>prepare-agent</goal>
-            </goals>
-            <configuration>
-              <excludes>
-                <exclude>hudson/plugins/jira/Messages*.class</exclude>
```

Comment:
why removing this?

Reply:
And that's why the travis file is related, I have `-P enable-jacoco` there. Generated files should not be part of the coverage report.